AmplifyTests/CategoryTests/DataStore/ModelFieldAssociationTests.swift

```diff
@@ -36,12 +36,13 @@ class ModelFieldAssociationTests: XCTestCase {
     }
 
     func testHasOneWithCodingKeys() {
```

Comment:
Can we have some test with the old code, to make sure it is backward compatible. Or keep this test as it is, and write another with the targetName fields.